c++/src/kj/refcount.h

```diff
@@ -134,10 +134,17 @@ class AtomicRefcounted: private kj::Disposer {
   KJ_DISALLOW_COPY(AtomicRefcounted);
 
   inline bool isShared() const {
+    return getRefcount() > 1;
+  }
+
+  inline uint getRefcount() const {
+    // This value can be useful for heuristic purposes, such as estimating the load on a resource.
+
 #if _MSC_VER
-    return KJ_MSVC_INTERLOCKED(Or, acq)(&refcount, 0) > 1;
+    static_assert(sizeof(uint) == sizeof(long), "unsafe implicit conversion");
```

Comment:
Sorry but I feel this is worse than what was here before. If there is some platform that MSVC targets where `long` happens to be 64 bits, this will fail to compile for no good reason.

Reply:
Okay, will remove it. (No such platform exists, though I suppose they might change that someday.)